slices/libc6.yaml

```diff
@@ -9,6 +9,8 @@ slices:
       /etc/ld.so.conf.d/*-linux-*.conf:
 
   libs:
+    essential:
+      - base-files_lib
```

Comment:
Suggestion: let's add a brief comment here explaining why we are including `base-files_lib`. Feel free to add a link to this PR as well so that in future, we can come back here to understand the rationale.

Reply:
Y good idea